### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by MichaelDeBoey**
BREAKING CHANGE: Requires Node 10+

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,24 +1,22 @@
...
-      script: npm run validate
-      # Travis does not support secrets on Windows yet so we test only external PRs where the secrets are not exposed
-      # https://travis-ci.community/t/current-known-issues-please-read-this-before-posting-a-new-topic/264
-      if: type = pull_request AND head_repo != repo
-    - stage: test and publish
-      name: test on linux and publish
-      os: linux
-      install: npm install
-      script: npm run validate
-      after_success: kcd-scripts travis-after-success
```
Comment (kentcdodds) [inline .travis.yml]: I think we want to keep the windows build somehow. Would you like to figure out how to make that possible?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -67,6 +40,7 @@ Please checkout the [the open issues][issues]
 Also, please watch the repo and respond to questions/bug reports/feature
 requests! Thanks!
 
+<!-- prettier-ignore-start -->
 [egghead]: https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github
-[all-contributors]: https://github.com/kentcdodds/all-contributors
-[issues]: https://github.com/kentcdodds/babel-plugin-macros/issues
+[issues]: https://github.com/kentcdodds/generator-kcd-oss/issues
```
Comment (kentcdodds) [inline CONTRIBUTING.md]:
```suggestion
[issues]: https://github.com/kentcdodds/babel-plugin-macros/issues
```

**Comment by kentcdodds** [comment]
I just noticed the bugs and feature requests links are broken

**Comment by MichaelDeBoey** [comment]
@kentcdodds Fixed in #147

**Pull request merged**